collections/_article/rendering-priorities-january-2024.md

```diff
@@ -0,0 +1,116 @@
+
+**1. Overhauled ParticlesProcessMaterial ([GH-79527](https://github.com/godotengine/godot/pull/79527))**
+
+**2. TextureRD ([GH-79288](https://github.com/godotengine/godot/pull/79288)) (official demo)**
+
+**3. Significant LightmapGI enhancements to both performance and quality**
+
+**4. Vertex compression optimization ([GH-81138](https://github.com/godotengine/godot/pull/81138))**
+
+**5. 2D HDR to support using 3D glow in 2D ([GH-80215](https://github.com/godotengine/godot/pull/80215))**
+
+**6. Optional ANGLE backend to support low end devices ([GH-72831](https://github.com/godotengine/godot/pull/72831))**
+
+For more information, check out the "Rendering, particles, and shaders" section of the [4.2 release blog post](https://godotengine.org/article/godot-4-2-arrives-in-style/#rendering-particles-and-shaders).
+
+
+## Priorities
+
+Below I list some of the top priorities identified by the [rendering team](https://godotengine.org/teams/#rendering). These priorities are:
+
+**1. D3D12 rendering driver ([GH-70315](https://github.com/godotengine/godot/pull/70315)) (merged for 4.3)**
+
+In December we merged support for a D3D12 rendering driver. This is the GPU backend of choice for Window's devices. Currently users need to [compile Godot from source](https://docs.godotengine.org/en/latest/contributing/development/compiling/compiling_for_windows.html#compiling-with-support-for-direct3d-12) to include support for D3D12, but we are investigating ways that we could provide D3D12 support out of the box.
+
+**2. RenderingDeviceDriver refactor ([GH-83452](https://github.com/godotengine/godot/pull/83452)) (merged for 4.3)**
+
+In preparing the D3D12 rendering driver, Pedro noticed that a significant amount of code needed to be shared between the Vulkan implementation of RenderingDevice and the D3D12 implementation of RenderingDevice. He proposed that we divide RenderingDevice into two pieces: 1) an API agnostic layer that contains all the shared code and 2) a thin layer that only includes API-specific code (the RenderingDeviceDriver). This has allowed us to save thousands of lines of duplicate code which will make porting to new platforms easier (e.g. Metal, WebGPU) and will make maintaining our current backend easier. 
+
+**3. Acyclic render graph optimization ([GH-84976](https://github.com/godotengine/godot/pull/84976)) (merged for 4.3)**
+
+This is both an optimization and a feature. The Acyclic Render Graph (ARG) is a tool to automatically record and re-order rendering commands in the RenderingDevice backend. This is extremely beneficial for 2 reasons: 1) it enables optimizations that are otherwise not possible in a general purpose renderer and 2) it simplifies the RenderingDevice API to make it much easier to use (this benefits us and users who use the RenderingDevice API directly). So far this PR has shown promising performance improvements on a range of devices and especially when many particle systems are in use. 
```

Comment:
```suggestion
This is both an optimization and a feature. The Acyclic Render Graph (ARG) is a tool to automatically record and re-order rendering commands in the RenderingDevice backend. This is extremely beneficial for 2 reasons: 1) it enables optimizations that are otherwise not possible in a general purpose renderer, and 2) it simplifies the RenderingDevice API to make it much easier to use (this benefits us and users who use the RenderingDevice API directly). So far this PR has shown promising performance improvements on a range of devices and especially when many particle systems are in use. 
```